explainer.md

```diff
@@ -88,21 +90,23 @@ only have accurate accounting when the process is hosting a single top level
 frame. As such, we return null anytime the process is hosting more than a single
 top level frame.
 
+For **totalJSHeapSize** and **usedJSHeapSize** we return the memory that corresponds to the execution context
```

Comment:
Just to be extra certain - we are able to detect when multiple top-level frames are being hosted in the same isolate, and will return null?

Reply:
or something along those lines - I just want it to be clear to users when we will return null

Reply:
I moved it in the text. We already say above that it will be null when there are multiple top level frame. If there is only one top level frame we will return the memory consumption of the given execution context - be it main frame isolate or a given worker.